ebos/eclwriter.hh

```diff
@@ -470,7 +478,7 @@ public:
     void endRestart()
     {}
 
-    const EclOutputBlackOilModule<TypeTag>& eclOutputModule() const
+    EclOutputBlackOilModule<TypeTag>& eclOutputModule() const
```

Comment:
I think I'd prefer if this were left to return a reference to `const`, and then we'd add a new function, e.g.,
```C++
EclOutputBlackoilModule<TypeTag>& mutableEclOutputModule()
{ return *eclOutputModule_; }
```
instead.  At least in that way, it's clear at the call site that we're performing a potentially mutating operation on the output module object.  You'd be forced to write something like
```C++
ebosSimulator_.problem().eclWriter()->mutableEclOutputModule().invalidateLocalData();
```
instead.

Reply:
changed it.